src/schemas/21-526EZ/schema.js

```diff
@@ -105,19 +105,42 @@ const fullNameDef = ((definitions) => {
 })(definitions);
 
 /**
- * Strip address lines from PCIU address def for use with treatment center addresses
+ * Modifies PCIU Address for use with treatments schema
  * @typedef {object} definitions
  * @property {object} pciuAddress
  * @param {definitions} definitions from the common schema definitions file
  * @returns {object} the treatmentCenterAddress schema object
  */
-const treatmentCenterAddressDef = ((definitions) => {
-  const treatmentAddressProperties = _.pick(
-    ['city', 'state', 'country'],
-    definitions.pciuAddress.properties
-  );
+const vaTreatmentCenterAddressDef = (({ pciuAddress }) => {
+  const { type, oneOf, properties } = pciuAddress;
+  return Object.assign({}, {
+    type,
+    oneOf: oneOf.map((obj) => _.cloneDeep(obj)),
```

Comment:
Why clone?

Reply:
🤷‍♂️ Ran into a nasty bug when I accidentally modified the base schema and now I'm paranoid. I'm happy to change this if you think it makes a difference.